distr_tracing/distr_tracing_rn/distr-tracing-rn-3-0.adoc

```diff
@@ -0,0 +1,86 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="distributed-tracing-rn-3-0"]
+= Release notes for {DTProductName} 3.0
+:context: distributed-tracing-rn-3-0
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
+[id="component-versions_distributed-tracing-rn-3-0"]
+== Component versions in the {DTProductName} 3.0
+
+[options="header"]
+|===
+|Operator |Component |Version
+|{JaegerName}
+|Jaeger
+|1.51.0
+
+|xref:../../otel/otel-release-notes.adoc[{OTELName}]
+|OpenTelemetry
+|0.89.0
+
+|{TempoName}
+|Tempo
+|2.3.0
+|===
+
+// Jaeger section
+[id="jaeger-release-notes_distributed-tracing-rn-3-0"]
+== {JaegerName}
+
+=== Deprecated functionality
+
+In Red Hat OpenShift distributed tracing 3.0, Jaeger and Elasticsearch are deprecated, and both are planned to be removed in a future release. Red Hat will provide critical and above CVE bug fixes and support for these components during the current release lifecycle, but these components will no longer receive feature enhancements.
```

Comment:
>  a future release

we don't know when that will happen (it's is not going to be in the next release but in some future release). Is it correctly reflected in the current phrasing?

Reply:
I believe so, but in any case, we'd better give sufficient warning to any customers that they might be.